### PR TITLE
Feature request template added

### LINKED ISSUE #162: [Feature] : Add issue template for feature request
@PiyushKalyanpy I think this a good feature to have and can organize the issues more

### PR DESCRIPTION
**Pull request by Tushar98644**
fixes #162 

@PiyushKalyanpy Kindly review....

### CODE REVIEW

**Review by lalitmeeq — COMMENTED** [review]
Hey @Tushar98644, can you please share some screenshots or the text preview of this? How is it looking?

And, instead of tagging the people in your comment just tag the people in the reviewers section. 
Check this [doc](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review#:~:text=Under%20your%20repository%20name%2C%20click,to%20their%20username%2C%20click%20Request.).

**Comment by Tushar98644** [comment]
> Hey @Tushar98644, can you please share some screenshots or the text preview of this? How is it looking?
> 
> And, instead of tagging the people in your comment just tag the people in the reviewers section. Check this [doc](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review#:~:text=Under%20your%20repository%20name%2C%20click,to%20their%20username%2C%20click%20Request.).

Screenshot 2023-05-27 at 2 57 15 PM

I don't think the repository permissions allow for a contributor to request a review , you need to be a collaborator or have write permissions , please confirm from your end

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Feature Request 💡
+description: Have any new idea or new feature ? Please suggest!
+title: "[Feature] <write a small description here>"
```
Comment (lalitmee) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:3]: Lets not add `[Feature]` here as we can always assign the labels to the issue.

Can we make a `label` of type `feature` and then automate that label to automatically attach with the feature in this template?

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+labels: ["enhancement", "goal: addition"]
```
Comment (lalitmee) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:4]: Add `feature` label here. Check the `labels` in the issues section. I have created a new `feature` label.

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      required: false
+  - type: checkboxes
+    id: no-duplicate-issues
+    attributes:
+      label: "Checklist"
+      options:
+        - label: "I have checked the existing issues"
+          required: true
+
+        - label: "I have read the [Contributing Guidelines]"
```
Comment (lalitmee) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:28]: Can we add the link to the `CONTRIBUTING.md` file here?

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+name: Feature Request 💡
+description: Have any new idea or new feature ? Please suggest!
+title: "[Feature] <write a small description here>"
+labels: ["enhancement", "goal: addition"]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A clear and concise description of any alternative solution or features you've considered.
```
Comment (lalitmee) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:10]: I think there should be two `textarea`, one for the feature description and other one for the alternative solutions of things they have tried.

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Comment by PiyushKalyanpy** [comment]
The changes requested were not done, so we acre closing this PR, please reopen the PR once you made the changes.

**Pull request closed without merging**